app/views/report/_reports.haml

```diff
@@ -0,0 +1,46 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+  = stylesheet_link_tag :report
+
+.col-md-9
+  .reports
+    %h1
+      = t("report.title")
+    - if @reports.empty?
+      %p
+        = t("report.unreviewed_reports", count: 0)
+    - @reports.each do |report|
+      - if report.item
+        .panel.panel-default
+          - username = report.user.username
+          .panel-heading
+            .reporter.pull-right
+              != t("report.reported_label", person: link_to(username, user_profile_path(username)))
+            .reason
+              %span.reason-label
+                = t("report.reason_label")
+              %span
+                = report.text
+          .panel-body
+            .content
+              = report_content(report)
+            .segment-selection
+              = button_to t("report.reported_user_details"),
+                user_search_path(admins_controller_user_search: {guid: report.reported_author.guid}),
+                class: "btn pull-left btn-info btn-small col-md-3 col-xs-12", method: :post
```

Comment:
The buttons should have a space in between:
![2017-04-28 15-32-56](https://cloud.githubusercontent.com/assets/10187586/25528868/5b3b7688-2c28-11e7-87c6-40d3f95832f3.png)